.circleci/WORKFLOW_MAPPING.md

```
@@ -0,0 +1,93 @@
+# CircleCI Workflow Mapping
+
+This document maps the original single-file workflows to the new multi-file structure.
+
+## Original Workflows in config.yml
+
+### 1. `commit` Workflow
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)